pkg/agent/apiserver/handlers/networkpolicy/handler.go

```diff
@@ -65,30 +66,34 @@ var mapToNetworkPolicyType = map[string]cpv1beta.NetworkPolicyType{
 }
 
 // Create a Network Policy Filter from URL Query
-func newFilterFromURLQuery(query url.Values) (*querier.NetworkPolicyQueryFilter, error) {
-	namespace := query.Get("namespace")
-	pod := query.Get("pod")
+func newFilterFromURLQuery(query url.Values) (*querier.NetworkPolicyQueryFilter, string, error) {
+	namespace, pod := query.Get("namespace"), query.Get("pod")
 	if pod != "" && namespace == "" {
-		return nil, fmt.Errorf("with a pod name, namespace must be provided")
+		return nil, "", fmt.Errorf("with a pod name, namespace must be provided")
 	}
-
 	strSourceType := strings.ToUpper(query.Get("type"))
 	npSourceType, ok := mapToNetworkPolicyType[strSourceType]
 	if strSourceType != "" && !ok {
-		return nil, fmt.Errorf("invalid reference type. It should be K8sNP, ACNP or ANP")
+		return nil, "", fmt.Errorf("invalid policy source type. Valid values are K8sNP, ACNP and ANP")
 	}
-
 	source := query.Get("source")
 	name := query.Get("name")
 	if name != "" && (source != "" || namespace != "" || pod != "" || strSourceType != "") {
-		return nil, fmt.Errorf("with a name, none of the other fields can be set")
+		return nil, "", fmt.Errorf("with a name, none of the other fields can be set")
+	}
+	// In case of getting policies applied to a specific Pod, the -n option is used to specify the
```

Comment:
I feel it's confusing that "-n" stands for the namespace of the Pod in one case and stands for the namespace filter of networkpolicies in another case.
Could we ask for "-p ns/pod" for the former case to be clear?

Reply:
@GraysonWu and I thought about this when we discussed how to fix this issue. The problem by going with "-p ns/pod" is that it doesn't make much sense for `-p` and `-n` to be used in conjunction anymore. Do you think we should disable this usage?

Reply:
Changed to "-p ns/pod" in the latest commit